src/lib/contextSet.js

```diff
@@ -35,6 +36,10 @@ class ContextSet {
     removeContextProp(id, key) {
         return removeUserProp(this.db, id, key);
     }
+    getPropValueList (key, value) {
```

Comment:
I am thinking of a more descriptive for this function as well, maybe call it listContexts or getContexts